.travis.yml

```diff
@@ -23,11 +23,11 @@ script: tools/travis-test.sh -p $PRESET
 
 after_failure:
         - cat `ls -1 -d -t apps/ejabberd/logs/ct_run* | head -1`/apps.ejabberd.logs/run.*/suite.log
-        - cat dev/mongooseim_node1/etc/ejabberd.cfg
-        - if [ -s dev/mongooseim_node1/log/crash.log ]; then cat dev/mongooseim_node1/log/crash.log; fi
-        - if [ -s dev/mongooseim_node2/log/crash.log ]; then cat dev/mongooseim_node2/log/crash.log; fi
-        - tail -100 dev/mongooseim_node1/log/ejabberd.log
-        - tail -100 dev/mongooseim_node2/log/ejabberd.log
+        - cat _build/mim1/rel/mongooseim/etc/ejabberd.cfg
```

Comment:
I think we can remove the line with printing the config but I'd leave the ones with logs, because it's easier and faster to take a quick look at this output, than navigate through S3, download file etc. :)